rust-tests/cbmc-reg/Pointers_Basic/fixme_from_raw.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+fn main() {
+    let address = 0x01234usize;
+    let ptr = address as *mut i32;
+    unsafe {
+        // random address cannot be dereferenced from memory
+        *ptr = 1;
+    };
```

Comment:
This new test is good but could use some comments. I have tried it using rustc (segmentation fault) and RMC (successful verification), so I think we want to include "fail" in its name if we expect RMC to fail verification.

Reply:
Clarified the comment.

Reply:
Add more comments explaining the RMC output. Also, a comment explaining that you cannot do this without being in unsafe region.

Reply:
How about now?